templates/taxi/car_detail.html

```diff
@@ -19,4 +19,14 @@
       <li>{{ driver.username }} ({{ driver.first_name }} {{ driver.last_name }})</li>
     {% endfor %}
   </ul>
+
+  {% if user not in car.drivers.all %}
+    <a href="{% url 'taxi:car-assign' pk=car.id %}" class="btn btn-primary">
+      Assign me to this car
+    </a>
+  {% else %}
+    <a href="{% url 'taxi:car-assign' pk=car.id %}" class="btn btn-danger">
+      Delete me from this car
+    </a>
```

Comment:
The links for assigning or removing the user from the car rely on the `assign_user_to_car` view. Ensure that this view is correctly implemented to handle these actions.